Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -610,3 +617,24 @@ void HexEditorWidget::drop_event(GUI::DropEvent& event)
         open_file(response.value().filename(), response.value().release_stream());
     }
 }
+
+ErrorOr<String> HexEditorWidget::make_display_string(DeprecatedString const& string)
```

Comment:
I still think this can take a StringView

Reply:
done 👍